packages/storybook/stories/Adapters/Adapters.stories.mdx

```diff
@@ -32,8 +31,6 @@ Teams Meeting information can be acquired through [Microsoft Graph](https://docs
 
 <Source code={CallAdapterExampleText} />
 
-<MessageBar>Note: The `displayName` in Call Composite can be a maximum of 256 characters.</MessageBar>
```

Comment:
Are these changes in this file intentional?

Reply:
Thanks, good catch.